packages/cli/src/index.ts

```diff
@@ -9,6 +9,20 @@ import Logger from './logger';
 //IMPORTANT: Using require instead of import here to avoid circular dependency issue rnv => @rnv/cli => rnv
 const { executeRnv } = require('rnv');
 
+const terminateProcesses = () => {
+    const { runningProcesses } = getContext();
+    try {
+        runningProcesses.forEach((p) => {
+            p.kill();
+        });
+    } catch (e) {
+        console.log(e);
+    }
+    runningProcesses.length = 0;
+
+    process.exit(0);
```

Comment:
you're calling process.exit here but then `logError(e, true)` afterwards. I'd remove it from here